### ISSUE TITLE
survey-vue: survey navigation becomes unresponsive if all survey fields are read-only

### ISSUE THREAD

**Issue by esune** [surveyjs/survey-library#2734]
### Are you requesting a feature, reporting a bug or asking a question?
Reporting a bug

### What is the current behavior?
When a multi-page survey displays only pre-filled, read-only fields, navigation breaks and it is not possible to  navigate away from the current survey page. The previous/next navigation buttons are not clickable.

### What is the expected behavior?
Regardless of how the fields are filled (user input or pre-populated), it should be possible to complete the survey successfully. Buttons should not become disabled.

### How would you reproduce the current behavior (if this is a bug)?
- Create a component like [this one](https://github.com/bcgov/issuer-kit/blob/master/issuer-web/src/components/DataCollection.vue) that will read a JSON configuration file, render the survey and pre-populate the fields and make them readOnly if a value is provided.
- Navigate the survey: after hitting the second page, it will be impossible to move forward or backward in the survey.

#### Provide the test code and the tested page URL (if applicable)

Vue Component Code: [see here](https://github.com/bcgov/issuer-kit/blob/master/issuer-web/src/components/DataCollection.vue)

Survey Configuration:
```json
{
  "pages": [
    {
      "title": "Page1 - page {pageno} of {pagecount}.",
      "questions": [
        {
          "name": "regNumber",
          "type": "text",
          "title": "Registration Number:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly": true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "Type",
          "type": "text",
          "title": "Type:",
          "placeHolder": "",
          "defaultValue": "Default Type",
          "isRequired": true,
          "readOnly": true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "regName",
          "type": "text",
          "title": "Registration Name:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly": true,
          "requiredErrorText": "Required Field"
        }
      ]
    },
    {
      "title": "Page2 - page {pageno} of {pagecount}.",
      "questions": [
        {
          "name": "streetAddress",
          "type": "text",
          "title": "Street Address:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "city",
          "type": "text",
          "title": "City:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "province",
          "type": "text",
          "title": "Province:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "country",
          "type": "text",
          "title": "Country:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "postalCode",
          "type": "text",
          "title": "Postal Code:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        }
      ]
    },
    {
      "title": "Page3 - page {pageno} of {pagecount}.",
      "questions": [
        {
          "name": "instructions",
          "type": "text",
          "title": "Instructions:",
          "placeHolder": "",
          "defaultValue": "N.A.",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "otherInstructions",
          "type": "text",
          "title": "Other Instructions:",
          "placeHolder": "",
          "defaultValue": "N.A.",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        }
      ]
    },
    {
      "title": "Page4",
      "questions": [
        {
          "name": "nok_FullName",
          "type": "text",
          "title": "Next of Kin -  Full Name:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "nok_StreetAddress",
          "type": "text",
          "title": "Next of Kin - Street Address:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "nok_City",
          "type": "text",
          "title": "Next of Kin - City:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "nok_Province",
          "type": "text",
          "title": "Next of Kin - Province:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "nok_Country",
          "type": "text",
          "title": "Next of Kin - Country:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "nok_PostalCode",
          "type": "text",
          "title": "Next of Kin - Postal Code:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        }
      ]
    },
    {
      "title": "Page5 - page {pageno} of {pagecount}.",
      "questions": [
        {
          "name": "registrationDate",
          "type": "text",
          "title": "Registration Date:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        },
        {
          "name": "registrationExpiryDate",
          "type": "text",
          "title": "Registration Expiry Date:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "defaultValue": "N.A.",
          "requiredErrorText": "Required Field"
        },
        {
          "name": "issueDate",
          "type": "text",
          "title": "Issue Date:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "defaultValue": "N.A.",
          "requiredErrorText": "Required Field"
        },
        {
          "name": "expirationDate",
          "type": "text",
          "title": "Expiration Date:",
          "placeHolder": "",
          "isRequired": true,
          "readOnly" : true,
          "defaultValue": "N.A.",
          "requiredErrorText": "Required Field"
        },
        {
          "name": "status",
          "type": "text",
          "title": "Status:",
          "placeHolder": "",
          "defaultValue": "Ready",
          "isRequired": true,
          "readOnly" : true,
          "requiredErrorText": "Required Field"
        }
      ]
    }
  ]
}
```

Setting up the example completely is not 100% straightforward as this is used as part of a bigger set of applications that are inter-dependant. I tried reproducing the issue with a vanilla JS implementation and minimal survey on plunkr [see here](https://plnkr.co/edit/dDWPGA7dBfqCdGtV), however I am unable to trigger the issue there. The problem is, however, consistent in my Vue survey.

The Vue component linked above has all of the code responsible for rendering the survey, I'll be happy to help setting up a more complex example if the component code + the configuration is not enough to reproduce and/or understand what the issue is.

### Specify your

* browser: Firefox Developer Edition
* browser version: 87.0b9
* browser: Chrome
* browser version: 89.0.4389.90
* surveyjs platform (angular or react or jquery or knockout or vue):  `vue`
* surveyjs version:  `survey-vue: ^1.8.26`

**Comment by andrewtelnov** [comment]
@esune Could you please help us to reproduce this issue in [Code SandBox example](https://codesandbox.io/s/surveyjs-vue-forked-icr20)? Your JSON, without your code works fine here.

Thank you,
Andrew

**Comment by esune** [comment]
@andrewtelnov I'll work on a setup either on Code SandBox or in a fork of my repo with all the code you need, I'll post it here once ready.

**Comment by esune** [comment]
@andrewtelnov I have put together a code sample [here](https://codesandbox.io/s/surveyjs-vue-forked-m77ol?file=/src/components/SurveyComponent.vue:7791-8408). However, in a similar way to when someone goes to the dentist with a toothache, the symptoms are not showing in my reproduced snippet... 😑 

I'll see if I can figure out another way of reproducing the issue so you can poke at it, if I am unsuccessful I guess I will close this.

**Comment by esune** [comment]
Aha! So there was a bit of a dependency mess going on and the actual version of `survey-vue` that triggers that issue is `1.8.12`.

I have updated the snippet to show the behaviour in case you're interested, but as far as I can tell version `1.8.26` fixes it. I have not tried newer versions because I was getting a compilation error about duplicated symbols (I think I saw an issue open here about that).

**Comment by andrewtelnov** [comment]
@esune Great!
I beleive the version we are going to release shortly v1.8.37 should not have a problem with duplicated symbols. I believe we fixed it in the current version v1.8.36, but I did not check it.
Anyway, I believe we can close this issue.

Thank you,
Andrew
